### ISSUE TITLE
'parentNode' is null or not an object – in IE<9

### ISSUE THREAD

**Issue by pmeissner** [kolber/audiojs#85]
In IE <9 the SWF does not load and I get the error message — 'parentNode' is null or not an object.

Line: 20, Char:345

**Comment by listao** [comment]
I'm having the same issue as well. Any update on this would be greatly appreciated. Thanks.

**Comment by listao** [comment]
Yeah, I see to be having the same issue. For me IE8 keeps reporting a the error 'parentNode' is null or not an object. In the uncompressed version, it is Line 584, character 5. Having traced it through the calls, it seems the issue is coming from the createPlayer() function call:

// Tracing this out in IE8 debugger reveals it returns a length of 0...
return wrapper.getElementsByTagName('audio')[0];
It can't seem to find the <audio> element. Weird thing is that like other users this works fine when I have the player (like the example pages), but when I have a string created, injected, then later initialize the player it throws this issue.
